Makefile

```diff
@@ -85,13 +88,20 @@ test: ## run test
 FORCE:
 
 # Build a binary from a cmd.
-bin/%: cmd/% FORCE
+$(COMMAND_BIN): bin/%: cmd/% FORCE
```

Comment:
This doesn't look right to me (although it's possible it's a gmake feature I'm not familiar with). I think you want a separate target for each binary. If you want to share the same rule, maybe you could use vpath: https://www.gnu.org/software/make/manual/make.html#Selective-Search